app.js

```diff
@@ -25,18 +28,21 @@ const sessionSettings = {
 // register all partials from directory.
 hbs.registerPartials("./views/partials");
 
-app.set("views", "./views");
+app.set("views", path.join(__dirname, "views"));
 app.set("view engine", "hbs");
 // app.use(favicon(path.join(__dirname, "/public/favicon.ico")));
 app.use(logger("dev"));
 app.use(methodOverride());
 app.use(session(sessionSettings));
+app.use(flash());
 app.use(bodyParser.json());
-app.use(bodyParser.urlencoded({ extended: true }));
+app.use(bodyParser.urlencoded({extended: true}));
 // app.use(multer({ dest: './uploads' }));
-app.use(express.static("./public"));
+app.use(express.static(path.join(__dirname, "public")));
+app.use(passport.initialize());
+app.use(passport.session());
 
-Router(app);
+registerRoutes(app, passport);
```

Comment:
Yes, I'm requiring a function. I changed it from a class so that it seems more appropriate.